src/buf.c

```diff
@@ -93,6 +95,15 @@ void rs_filebuf_free(rs_filebuf_t *fb)
 }
 
 
+void rs_get_filesize(FILE *f, rs_long_t *size)
```

Comment:
fseek and ftell are defined in buf.c, I didn't want to move (or duplicate) those defines...